app/uk/gov/hmrc/gform/eval/smartstring/SmartStringEvaluatorFactory.scala

```diff
@@ -193,6 +200,11 @@ class RealSmartStringEvaluatorFactory() extends SmartStringEvaluatorFactory {
     new SmartStringEvaluator {
       override def apply(s: SmartString, markDown: Boolean): String =
         s.rawValue(l)
+
+      override def evalEnglish(s: SmartString, markDown: Boolean): String = {
+        val englishImplicitLangADT: LangADT = LangADT.En
+        self.noForm(englishImplicitLangADT)(s, markDown)
```

Comment:
I think this is better:
```suggestion
        s.rawValue(LangADT.En)
```